spec.html

```diff
@@ -32156,6 +32156,33 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-time-zone-identifiers">
+        <h1>Time Zone Identifiers</h1>
+
+        <p>
+          Time zones in ECMAScript are represented by <dfn variants="time zone identifier">time zone identifiers</dfn>, which must be Strings composed entirely of characters from the Unicode Basic Latin block.
+          Time zone identifiers are compared using ASCII case-insensitive comparisons.
+          Time zones supported by an ECMAScript implementation may be <dfn variants="available named time zone">available named time zones</dfn>, represented by the [[Identifier]] field of the Time Zone Identifier Records returned by AvailableNamedTimeZoneIdentifiers, or <dfn variants="offset time zone">offset time zones</dfn>, represented by Strings for which IsTimeZoneOffsetString returns *true*.
+        </p>
+        <p>
+          A <dfn variants="primary time zone identifiers">primary time zone identifier</dfn> represents the preferred identifier of an available named time zone, while a <dfn variants="non-primary time zone identifiers">non-primary time zone identifier</dfn> is an alias for the same available named time zone.
+          Each available named time is represented by exactly one primary time zone identifier and zero or more non-primary time zone identifiers.
```

Comment:
```suggestion
          Each available named time zone is associated with exactly one primary time zone identifier and zero or more non-primary time zone identifiers.
```

I would probably combine this with the previous paragraph.

Reply:
Should we say something about the relationship in the other direction? Something like "Each primary time zone identifier and each non-primary time zone identifier is associated with no more than one available named time zone". Also, maybe we need to dfn "time zone identifier" so we don't have to say "primary time zone identifier or non-primary time zone identifier".

Reply:
Good idea. I used the text below for the reverse direction. Let me know if you want to see something different.

> Each primary or non-primary time zone identifier is associated with exactly one available named time zone.